README.md

```diff
@@ -53,14 +53,16 @@ The following table lists the configurable parameters of the `kubernetes-externa
 | Parameter                            | Description                                                  | Default                                                 |
 | ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------- |
 | `env.AWS_REGION`                     | Set AWS_REGION in Deployment Pod                             | `us-west-2`                                             |
+| `env.AWS_ACCESS_KEY_ID`              | Set the AWS access key associated with an IAM user or role.  | none                                                    |
```

Comment:
Should we extend the description to communicate that IAM roles are used by default if these aren't specified?